python/smt_switch/pysmt_frontend/pysmt_solver.py

```diff
@@ -199,6 +199,22 @@ def _reset_assertions(self):
     SWITCH_SOLVERS['btor'] = SwitchBtor
 
 
+if 'bitwuzla' in  ss.solvers:
+    logics_params = dict(
+        quantifier_free=[True],
+        arrays=[True, False],
```

Comment:
What does `[True, False]` mean in this context? Same for `uninterpreted` and `floating_point` below?

Reply:
Basically this is just an easy way of generating all of the supported logics for the solver. The list for each `logics_params `entry is the available settings for that option, with the default being [False]. Every possible combination of these options is then used to generate the list of supported logics.
For example, `quantifier_free=[True], bit_vectors=[True], uninterpreted=[True, False]` would result in supported logics of `QF_BV, QF_UFBV`.

Reply:
So what does the `False` mean in the `[True,False]` part of, say, `uninterpreted`?

Reply:
We want to generate every supported logic for this solver, which in this case includes both QF_BV and QE_UFBV, the `[True,False]` for uninterpreted says that the solver supports both. We are generating the list of supported logics from the every possible combination of these True/False lists.